packages/react/src/Link/Link.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2021 Gemeente Utrecht
+ * Copyright (c) 2021 Robbert Broersma
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Link as CommunityLink } from '@utrecht/component-library-react'
+import clsx from 'clsx'
+import { AnchorHTMLAttributes, ForwardedRef, forwardRef, PropsWithChildren } from 'react'
+
+export type LinkColor = 'default' | 'black' | 'white'
+
+export type LinkVariant = 'default' | 'bold'
+
+export interface LinkProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
+  href: string
+  placeholder?: string
+  external?: boolean
+  rel?: string
+  target?: string
+  color?: LinkColor
+  variant?: LinkVariant
+}
+
+export const Link = forwardRef(
+  (
+    { children, href, placeholder, variant, color, className, ...restProps }: PropsWithChildren<LinkProps>,
+    ref: ForwardedRef<HTMLAnchorElement>,
+  ) => (
+    <CommunityLink
+      href={placeholder ? undefined : href}
```

Comment:
Don't really get this logic. First, in the Utrecht Link `placeholder` is a boolean. Here it is a string. Even then, I don't get it :) Since `href` is required so assuming the users needs to set that prop, then this logic would not be needed right? Why would be prefer an `undefined` over a defined `href`?

Reply:
`undefined` will disable the link, which is what the Utrecht link placeholder is for.